types/rusha/index.d.ts

```diff
@@ -4,17 +4,15 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // Minimum TypeScript Version: 4.0
 
-/// <reference types="node" />
-
 interface Hash {
-    update(value: string|number[]|ArrayBuffer|Buffer): Hash;
+    update(value: string|number[]|ArrayBuffer): Hash;
```

Comment:
`ArrayBuffer` should not be considered a _super type_ of `Uint8Array` (or `Buffer`). While `Uint8Array`'s type currently is a superset of the members of `ArrayBuffer`, that could change. Based on how `rusha` [checks for typed arrays](https://github.com/srijs/rusha/blob/7dc2021195219e54b2696af524141696f35694dd/src/conv.js#L99-L101), the following change is probably safer:

```suggestion
    update(value: string | number[] | ArrayBuffer | { buffer: ArrayBuffer }): Hash;
```